File-Storage-Security/Deployment/aws-python-deploy-to-all-existing-s3/deploy.py

```diff
@@ -173,7 +171,20 @@ def deploy_storage(kms_arn, region, bucket_name):
             'mode': 'standard'
         }
     )
-
+    # gather cloud one ext id
+    r = http.request(
+        "GET",
+        stacks_api_url+"external-id",
+        headers={
+            "Authorization": "ApiKey " + ws_api,
+            "Api-Version": "v1",
+        },
+    )
+    try:
+        ext_id = json.loads(r.data.decode("utf-8"))['externalID']
+    except json.decoder.JSONDecodeError:
+        time.sleep(1)
+        ext_id = json.loads(r.data.decode("utf-8"))['externalID']
     # set fss api doc parameters
     ExternalID = {"ParameterKey": "ExternalID", "ParameterValue": ext_id}
```

Comment:
I'm also no specialist in Python, but the `ext_id`, even inside of a `try` `except` statements, both are part of the `deploy_storage` function, so that should work... any thoughts @JustinDPerkins ?

Reply:
Correct @felipecosta09. Tested the script yesterday and all was well.